.github/actions/commit-generated-files/action.yml

```diff
@@ -7,7 +7,7 @@ runs:
         if [ ! -z "$(git status --porcelain)" ]; then  
           git fetch --depth=1
           git checkout ${{ github.head_ref }}
-          git add 'packages/atomic/**/*.d.ts' 'packages/atomic-react/src/components/stencil-generated/*'
+          git add 'packages/atomic/**/*.d.ts' 'packages/atomic-react/src/components/stencil-generated/*' 'packages/atomic-angular/projects/atomic-angular/src/lib/stencil-generated/*'
```

Comment:
Kind of unrelated to this change, but I realized I forgot to add the necessary path here to have the PR bot auto regenerate atomic-angular files if needed.